device-base/Dockerfile.armv7hf.rpi.tpl

```diff
@@ -4,6 +4,13 @@ LABEL io.resin.device-type="#{DEV_TYPE}"
 
 RUN echo "deb http://archive.raspbian.org/raspbian #{SUITE} main contrib non-free rpi firmware" >>  /etc/apt/sources.list
 RUN apt-key adv --keyserver pgp.mit.edu  --recv-key 0x9165938D90FDDD2E
+RUN apt-get update && apt-get install -y --no-install-recommends wget \
+	&& wget http://archive.raspberrypi.org/debian/raspberrypi.gpg.key \
```

Comment:
This is totally unsecure. You're downloading a key over HTTP and then trusting it.